server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -115,25 +196,14 @@ public class PodSpec {
         public Set<File> includePaths = new HashSet<>();
```

Comment:
We don't want to use Set here, as it's unordered, and for compile flags, it needs to be controlled.
E.g. you don't want to switch order of include paths, as that might get you a different header file each compile run.

Reply:
Good point! I've changed to a LinkedHashSet (ie a set with a predictable iteration order based on insertion order)